packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -35,6 +36,18 @@ export type ToastProps = {
    * Content of the Toast message
    */
   content: React.ReactNode;
+  /**
+   * Passes props to content
+   */
+  contentProps?: React.ComponentPropsWithoutRef<'div'>;
+  /**
+   * Passes props to status area
+   */
+  statusAreaProps?: React.ComponentPropsWithoutRef<'div'>;
+  /**
+   * Svg icon shown on the left of toast message.
+   */
+  startIcon?: React.ReactNode;
```

Comment:
has anybody asked for support for adding custom icons?

Reply:
alert was refactored completely, and it started from a user request.

if someone requests it in toast, we can revisit this, but right now the goal is to just be able to add class names to existing dom elements

Reply:
Fine

Reply:
Just curious, is there a downside to supporting a custom start icon? Like if there is no downside, should we leave it as it is instead of spending time undoing it?

Reply:
Removed icon prop.